### DISCUSSION TITLE
Multiple "profiles"-like settings for sets of API keys.

### DISCUSSION THREAD

**Discussion by ErikMeinders** [cline/cline#472] [Feature Requests]
**As** a Developer
**I want** the costs of API usage to be related to the correct project
**So that** I can charge the individual projects correctly.

Possible solution:
Today, I use multiple API keys with the same API provider. Currently, I can not see which key is used. I'd like to be able to create named sets of API keys that I can select from, much like how I select a named profile in VSCode to use a particular set of Extensions.

**Comment by wbdb** [comment]
Wouldn't it be better to simply have several profiles comparable to RooCode, where you could store identical providers, other models and other API keys? It would be particularly relevant for me to be able to create several OpenRouter models.

https://github.com/cline/cline/discussions/1711

**Comment by dcbartlett** [comment]
We are looking into adding a feature similar to this.  Stay tuned!
